src/main/java/com/kakao/cafe/web/UserController.java

```diff
@@ -32,12 +37,12 @@ public UserController(UserService userService) {
     @GetMapping("/user/sign-up")
     public String joinForm() {
         logger.info("User in joinForm");
-        return "/user/form";
+        return "user/form";
     }
 
     @PostMapping("/user/sign-up")
```

Comment:
POST /user 만으로도 회원 가입을 의미합니다. 
```suggestion
    @PostMapping("/user")
```

Reply:
url 매핑에 대해서 조금 더 공부해보겠습니다!